test/tools/ossfuzz/proto_to_yul.h

```diff
@@ -0,0 +1,27 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <cstdint>
+#include <cstddef>
+#include <string>
+
+namespace yul_fuzzer {
+class Function;
+
+std::string FunctionToString(const Function &input);
```

Comment:
```suggestion
std::string FunctionToString(Function const& _input);
```

Reply:
I'm afraid I don't understand why in lining is necessary? The definition is in the `cpp` file that can be independently linked, right?

Reply:
You are right!